README.md

```diff
@@ -9,8 +9,9 @@ To reliably recreate any issue experienced in CI or Prod you should continue to
 Important: Keep in mind that docker is just used to improve the developer experience.
 
 1. Clone the project `git clone https://github.com/elifesciences/search.git`
-2. Rename `/dev/config.php.dist` on local to `/dev/config.php` 
-3. Run `docker-compose -f dev/docker-compose.yaml up --build`
+2. Rename `/dev/config.php.dist` on local to `/dev/config.php`
+3. Run `mkdir logs && sudo chmod 777 ./logs`
```

Comment:
We could add logs to dev folder with .gitkeep if needed? This would avoid the need to have this step but will try this out.